specification/contosowidgetmanager/data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/widgets.json

```diff
@@ -152,7 +152,7 @@
             "description": "The request has succeeded.",
             "headers": {
               "Operation-Location": {
-                "description": "The location of an instance of OperationStatus",
+                "description": "The location for monitoring the operation state.",
```

Comment:
Why did this description change, when I don't see a similar change in the tsp?

For my learning, is this considered a "breaking change"?

Reply:
Is this considered a "breaking change"?

Reply:
No, field documentation would not be a breaking change under any definition I am aware of.